lusSTR/str_markers.json

```diff
@@ -269,7 +350,11 @@
         "ReverseCompNeeded": "No",
         "LUS": "TAGA",
         "Sec": "",
-        "Tert": ""
+        "Tert": "",
+        "Foren_5": 17,
+        "Foren_3": 7,
+        "Power_5": "",
+        "Power_3": ""
```

Comment:
In some places empty strings are used, in other places 0s are used. It should be consistent, and judging from the `full_seq_to_uas` code it should probably be zeros.

Reply:
ahhh actually these are instances where the locus isn't in PowerSeq... I meant to address this, so let's hold off on merging until I incorporate this.

Reply:
I'm just going to remove the 'Power_5' and 'Power_3' from these loci.